src/audio-worklet/free-queue/example/js/freequeue.mjs

```diff
@@ -0,0 +1,170 @@
+/**
+ * A shared storage for FreeQueue operation backed by SharedArrayBuffer.
+ *
+ * @typedef SharedRingBuffer
+ * @property {Uint32Array} states Backed by SharedArrayBuffer.
+ * @property {number} bufferLength The frame buffer length. Should be identical
+ * throughout channels.
+ * @property {Array<Float32Array>} channels The length must be > 0.
+ * @property {number} channelCount same with channelData.length
+ */
+/**
```

Comment:
What's the difference between the comment between L1~L10 and the one at L11~L15?